corehq/apps/app_manager/static/app_manager/js/forms/advanced/case_config_ui.js

```diff
@@ -207,7 +207,7 @@ hqDefine('app_manager/js/forms/advanced/case_config_ui', function () {
                     $home.on('textchange', 'input', self.change)
                         // all select2's are represented by an input[type="hidden"]
                         .on('change', 'select, input[type="hidden"]', self.change)
-                        .on('click', 'a:not(.header)', self.change)
+                        .on('click', ':not(.hq-help) > a:not(.header)', self.change)
```

Comment:
i feel in general this should be a whitelist vs how we have it now. @orangejenny thoughts?
not blocking comment.

Reply:
yeah, I agree with that